tests/tape/tapes/test_cv_param_shift.py

```diff
@@ -320,7 +320,7 @@ def test_no_poly_xp_support(self, mocker, monkeypatch, caplog):
             tape.jacobian(dev, method="analytic")
 
         spy1.assert_not_called()
-        spy2.assert_called()

```

Comment:
How come this had to change? :thinking:

Reply:
Ah - previously it was `parameter_shift_second_order` method that caught whether the device supported PolyXP or not, and decided where to dispatch the gradient method to. This was moved up a method to `parameter_shift`, however.

I'll add an additional spy to show that `parameter_shift` is being called, _but_ `numeric_pd` is doing the computation.

Reply:
Done!